### DISCUSSION TITLE
xRechnung and LeitwegID

### DISCUSSION THREAD

**Discussion by SR84** [stephanstapel/ZUGFeRD-csharp#330] [Q&A] [answered]
I would like to create an xRechnung-compatible PDF/XML file. Unfortunately, I am not clear on how to set the LeitwegID. Can someone help me with this?

Responses can also be in German

**Comment by stephanstapel** [comment, marked as answer]
This one goes iinto BT-10:
https://de.wikipedia.org/wiki/Leitweg-ID

i.e. property ReferenceOrderNo

**Comment by SR84** [comment]
Thank you for the quick response.

So, is this the wrong approach?
desc.SetBuyerElectronicAddress(LeitWegId, s2industries.ZUGFeRD.ElectronicAddressSchemeIdentifiers.LeitwegID)

i have to put it here:
desc.ReferenceOrderNo = LeitWegId

**Comment by stephanstapel** [comment]
SetBuyerElectronicAddress() is for the buyer. ReferenceOrderNo should be for the seller afaik

**Reply by SR84** [reply]
Okay, got it. Then I need SetBuyerElectronicAddress(). I only know the LeitwegID of the invoice recipient.

Thank you for the quick information.
